src/fmu/sumo/sim2sumo/_special_treatments.py

```diff
@@ -0,0 +1,18 @@
+"""Special treatment of some options used in ecl2df"""
+from ecl2df.common import convert_lyrlist_to_zonemap, parse_lyrfile
+
+
+def final_touches(options):
```

Comment:
can we rename this to something more descriptive?

Reply:
Done